### ISSUE TITLE
Loss doesn't decrease and AUC doesn't improve on two-class classification task. 

### ISSUE THREAD

**Issue by davidsonic** [rusty1s/pytorch_geometric#409]
## ❓ Questions & Help

I tried the recommended tutorial on official website: https://towardsdatascience.com/hands-on-graph-neural-networks-with-pytorch-pytorch-geometric-359487e221a8 and I encountered  almost the same situation when I tried to train GCNConv model on my own dataset, where the goal is to classify between two classes. The loss doesn't decrease and AUC doesn't improve, even though I tried different learning rates, checked that data/label pairs are matched and gradient flows seemed ok. 

![image](https://user-images.githubusercontent.com/15846113/59470390-62268280-8dec-11e9-99d9-6a36cb0a0fa7.png)

![image](https://user-images.githubusercontent.com/15846113/59470230-f2b09300-8deb-11e9-814c-ff20b5c47256.png)

I also verified that model params were updated (conv bias  change from 0 to other values). Has anyone encountered this problem before? Any suggestions?

**Comment by rusty1s** [comment]
It is quite hard to help you with this one. You should simplify your model (and evaluate a simply MLP baseline) to see if this fixes your issues.

**Comment by davidsonic** [comment]
Thanks for your response. I tried MLP as following: load data using geometric.DataLoader, and then performed global_average_pooling across nodes in the graph, which gives me a 1xd feature of the whole graph. This feature was fed into several layers of FC with ReLU nonlinearity. The cross entropy loss calculated with groundtruth still doesn't give me any luck (binary prediction acc around 50%). Any suggestions?

**Comment by rusty1s** [comment]
Mh, maybe this is a data issue? How do your input features look like?

**Comment by davidsonic** [comment]
I used pretrained resnet50 released by torchvision model zoo, and it looks like this on my dataset:
![image](https://user-images.githubusercontent.com/15846113/59526739-833faf80-8e8e-11e9-9133-dc2f59f27131.png)
![image](https://user-images.githubusercontent.com/15846113/59526759-9783ac80-8e8e-11e9-82b7-1e23d71bd8f5.png)

**Comment by davidsonic** [comment]
Got it working right now! The issue has more to do with how I frame the problem and loss rather than coding issue. Close it for now.
